Homework-10/task_2.py

```diff
@@ -0,0 +1,10 @@
+def find_maximum(*numbers):
+    if not numbers:
+        return None
+    return max(numbers)
```

Comment:
ზოგადად, ფუნქციას იმისთვის ვქმნით რომ შიგნით ლოგიკა დავწეროთ და ფუნქციაში შიგნით მხოლოდ მზა ფუნქცია თუ გამოვიყენეთ 1 ხაზისთვის ამ ფუნქციის შექმნა არამართებულია, გთავაზობ ალტერნატივას:
def my_max(*args):
_max = args[0]
for i in args:
if type(i) == int or type(i) == float:
if i >_max:
_max = i
return _max

print(my_max(1,2,"car",-7.5,25,15,27))